sources/core/Stride.Core/PlatformFolders.cs

```diff
@@ -182,22 +184,27 @@ private static string GetApplicationTemporaryDirectory()
         [NotNull]
         private static string GetApplicationBinaryDirectory()
         {
-            return FindCoreAssemblyDirectory(GetApplicationExecutableDiretory());
+            var executablePath = GetApplicationExecutableDirectory();
+#if STRIDE_PLATFORM_ANDROID
+            return executablePath;
+#else
+            return FindCoreAssemblyDirectory(executablePath);
+#endif
         }
 
-        private static string GetApplicationExecutableDiretory()
+        private static string GetApplicationExecutableDirectory()
         {
 #if STRIDE_PLATFORM_WINDOWS_DESKTOP || STRIDE_PLATFORM_MONO_MOBILE || STRIDE_PLATFORM_UNIX
             var executableName = GetApplicationExecutablePath();
             if (!string.IsNullOrEmpty(executableName))
             {
                 return Path.GetDirectoryName(executableName);
             }
-    #if STRIDE_RUNTIME_CORECLR
+#if STRIDE_RUNTIME_CORECLR
```

Comment:
Any reason you adjusted the tab?
I don't mind much, but still they were helpful to understand the #ifdef structure.

Reply:
Anyway, not a big deal, will merge as is
4.1 looks OK for that.

Reply:
Odd, that file was copied over from Basewq's backport but the tabs are present in their commit.

https://github.com/Basewq/stride/blob/2e387ef78f1a4ba7211564e08249c2b9ee2f3642/sources/core/Stride.Core/PlatformFolders.cs

Might be Visual Studio weirdness, I'll put the tab back in and sneak it into the DebugText Color fix backport for 4.0.

Reply:
No need to bother, `master` is OK